recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "63.4.1" %}
+{% set version = "65.4.0" %}
```

Comment:
Can we build the latest v65.5.0 https://github.com/pypa/setuptools/blob/v65.5.0/CHANGES.rst#v6550

Reply:
Got it, will do after merging this one!